content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -250,12 +250,54 @@ spec:
 TODO: Test and explain how to use additional non-K8s secrets with an existing service account.
 -->
 
-## Service Account Volume Projection
+## Service Account Token Volume Projection
 
-Kubernetes 1.11 and higher supports a new way to project a service account token into a Pod.
-You can specify a token request with audiences, expirationSeconds. The service account token
-becomes invalid when the Pod is deleted. A Projected Volume named
-[ServiceAccountToken](/docs/concepts/storage/volumes/#projected) requests and stores the token.
+{{< note >}}
+**Note:** This ServiceAccountTokenVolumeProjection is __beta__ in 1.12 and
+enabled by passing the following flags to the API server:
+
+```
+* --service-account-issuer
+* --service-account-signing-key-file
+* --service-account-api-audiences
+```
```

Comment:
This code block is formatted a bit weirdly. Looks like you want bullets with the flags code formatted? In which case try this instead:

```
* `--service-account-issuer`
* `--service-account-signing-key-file`
* `--service-account-api-audiences`
```